lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -569,6 +570,21 @@ constructTransactionFragment ins outs mkWit = do
         ins outs mkWit
         (const mempty)
 
+constructDelegationCertificateTx
+    :: Monad m
+    => PoolId
+    -> ChimericAccount
+    -> (ByteString -> ByteString) -- ^ Signature by the chimeric account
+    -> [(TxIn, TxOut)]
+    -> [TxOut]
+    -> (Address -> m (Hash "sigData" -> TxWitness))
+    -> m (Hash "Tx", SealedTx)
+constructDelegationCertificateTx pool account auth ins outs mkWit = do
+    constructGenericSignedTx FragmentDelegation
+        (putStakeCertificate pool account)
+        ins outs mkWit
+        (("\SOH" <>) . auth)
```

Comment:
I didn't know what this was, but it signifies account vs multi-account signature:
```
ACCOUNT-SIG       = (%x01 SINGLE-ACNT-SIG) / (%x02 MULTI-ACNT-SIG)
```
from [format.abnf](https://github.com/input-output-hk/chain-libs/blob/master/chain-impl-mockchain/doc/format.abnf)